src/main/resources/test_files/runtime-spark/DataFrames/GroubyClauseTypeCheckError.jq

```diff
@@ -1,4 +1,4 @@
-(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:3:COLUMN:0:" :)
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:3:COLUMN:25:" :)
 for $j as integer in parallelize((1 to 10))
 group by $k as string := $j
 return $k
```

Comment:
Tiny naming note here for consistency. All other files are called Group[B]yClause...

Reply:
DONE